groupcachepb/groupcache.proto

```diff
@@ -31,6 +31,16 @@ message GetResponse {
   google.protobuf.Timestamp ttl = 3;
 }
 
+message ContainRequest {
+  string group = 1;
+  string key = 2; // not actually required/guaranteed to be UTF-8
+}
+
+message ContainResponse {
+  double minute_qps = 1;
+  bool exists = 2;
```

Comment:
nit: I don't think ordering actually matters here, but intuitively it seems that exists would be the field I'd want to see first, similar to "value" in the GetResponse, and the minute_qps field is secondary to that.

Reply:
Done